### DISCUSSION TITLE
4mmc/Mephedrone + Nitrous Oxide

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#158] [Drug Interactions]
Mephedrone + Nitrous Oxide - Low risk + synergy

There seems to be little reason to have concern about this combination.

Initially, there seemed to be some concern possibly of additional oxidative stress caused by NO2 in combination with the Mephedrone. This does not seem to be of enough concern to note however I am inquiring about this more. Leaving this here and sharing my research so far.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9738214/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4066238/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2617738/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7968496/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1821130/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4422847/

**Comment by zelixir25** [comment]
I agree with your conclusion that this is low risk

**Reply by utaninja** [reply]
Thank you!

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/342. Thanks @zelixir25!
